WcaOnRails/config/locales/de.yml

```diff
@@ -2569,48 +2569,71 @@ de:
       payment_successful: >-
         Deine Zahlung war erfolgreich und deine Zahlung sollte unten als bezahlt
         markiert sein.
+      #original_hash: 92dde77
+      payment_processing: 'Deine Zahlung wurde bestätigt, aber die Transaktion steht noch aus.'
+      #original_hash: 62d7351
+      button_text: Zahle jetzt!
       hints:
         #original_hash: bf3a39e
         donation: Zusätzliche Spenden gehen nur an das Organisationsteam
-        #original_hash: 16cf92d
-        card_information: >-
-          Deine Karteninformationen werden nicht an die WCA Website übertragen.
-          Sie werden verschlüsselt an Stripe übertragen, welches die Abwicklung
-          der Zahlung übernimmt und an uns weiterleitet.
+        #original_hash: f7fa263
+        payment_button: >-
+          Deine Zahlungsinformationen gelangen niemals zur WCA Website. Sie
+          werden verschlüsselt an Stripe übertragen, welches den Zahlungsprozess
+          abwickelt und an uns weiterleitet.
       labels:
         #original_hash: 4de825f
         entry_fees: Eintrittsgebühren
+        #original_hash: dc9d458
+        fees_paid: Bezahlt
+        #original_hash: cc632b5
+        fees_remaining: Ausstehend
         #original_hash: 2c09302
         donation: Spende
-        #original_hash: 184150d
-        card_information: Karteninformationen
-        #original_hash: bc8091a
-        cardholder_name: Name des Karteninhaber
         #original_hash: 919710d
         show_donation: >-
           Ich würde dem Organisationsteam gerne eine zusätzliche, optionale
           Spende zukommen lassen
+        #original_hash: 4930c06
+        payment_information: Zahlungsinformationen
+        #original_hash: de0fc95
+        payment_button: Zahle deine Gebühren mit Stripe
+        #original_hash: f94d00f
+        ajax_error: WCA Server Error
+        #original_hash: e62c56c
+        stripe_error: Stripe Error
```

Comment:
Ist "Error" als Anglizismus okay in dem Zusammenhang?

Reply:
Geändert zu Fehler, habe das Wort Fehler für Error auch an anderer Stelle wiedergefunden, dann haben wir es einheitlich.